core-errors/src/lib.rs

```diff
@@ -84,7 +86,9 @@ pub enum MessageError {
         existential_deposit
     )]
     InsufficientValue {
+        /// Message's value
```

Comment:
Dots at the end of each doc-comment, please

Reply:
I wondering why are they there

Reply:
Doc comments good manners)
The end of such docs life is auto-generated docs website and it's more clear and beautiful when each sentence has its finish